public/js/twenty.js

```diff
@@ -25,6 +29,31 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   }
 
+  const notify = () => {
+    if (Notification.permission !== 'granted') {
+      return;
+    }
+
+    let message;
+
+    if (mode === 'twenty') {
+      if (!stop) {
+        // can't see the notification if they're looking away!
+        return;
+      }
+
+      message = 'Time to look away!';
```

Comment:
might be better to have these strings as data like `limits`, maybe a copy pasted `notifications` structure? then you could just look it up and handle everything in one way

Reply:
Done, I also made it not notify you on startup / if you manually change the mode by clicking